SofaKernel/framework/sofa/simulation/CMakeLists.txt

```diff
@@ -131,7 +131,10 @@ set(SOURCE_FILES
     InitTasks.cpp
 )
 
-
+# is this a compiler/linker version specific problem?
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread" )
```

Comment:
[](https://cmake.org/cmake/help/v3.4/command/target_compile_options.html)

Reply:
-pthread should be passed as linking  options:
target_link_options(${PROJECT_NAME} PUBLIC "-pthread")
it doesn't work because the target_link_options is  available only starting from CMake 3.13